### ISSUE TITLE
Workbook corrupts when copying worksheet that uses dynamic array formula

### ISSUE THREAD

**Issue by craigbrown** [EPPlusSoftware/EPPlus#459]
Newer versions of Excel have a feature called [dynamic array formulas](https://support.microsoft.com/en-us/office/dynamic-array-formulas-and-spilled-array-behavior-205c6b06-03ba-4151-89a1-87a7eb36e531). These are similar to normal array formulas, but they fill the cells automatically without needing to use ctrl+shift+enter.

Excel marks the first cell of a dynamic array formula using the `cm` attribute on the `c` cell element:
```xml
<c r="A12" s="1" cm="1">
    <f ref="A12:J12" t="array">TRANSPOSE(B1:B10)</f>
    <v>1</v>
</c>
```

This `cm` value references an index in the `xl\metadata.xml` file in the package, which specifies that this is a dynamic array formula.

**The Bug**
1. Copy a sheet containing a dynamic array formula into a new workbook using EPPlus.
2. Open the new workbook in Excel. Excel shows an error message saying the workbook is corrupt and asks to repair it.

I think this because when the sheet is copied, the `metadata.xml` file isn't copied but the cell keeps its `cm` value. The fact that the cell has a `cm` value but there is no `metadata.xml` file means Excel marks the workbook as corrupt.

I think this could either be fixed by copying the relevant metadata values with the sheet, or by just removing the `cm` attributes when the sheet is copied. The latter is probably the simplest solution, and doesn't lose much functionality - it looks like it just downgrades the dynamic array formulas to be legacy array formulas.

Thanks for your help as always!

**Unit Test**
```csharp
[TestMethod]
public void CopyWorksheetWithDynamicArrayFormula()
{
    var openPath = @"C:\path\to\TestDynamicArrayFormula.xlsx";
    var savePath = @"C:\path\to\TestDynamicArrayFormula_" + System.DateTime.Now.ToString("yyyyMMddTHHmmss") + ".xlsx";
    
    using (var pck = new ExcelPackage(new FileInfo(openPath)))
    {
        using (var newPck = new ExcelPackage())
        {
            newPck.Workbook.Worksheets.Add("Sheet1", pck.Workbook.Worksheets["Sheet1"]);
            newPck.SaveAs(new FileInfo(savePath));
        }
    }
    
    Assert.Inconclusive("Now try to open the file in Excel - does it tell you the file is corrupt?");
}
```
[TestDynamicArrayFormula.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/6994460/TestDynamicArrayFormula.xlsx)

**Comment by JanKallman** [comment]
Yes, EPPlus do not yet fully support array formulas and meta data. I'll have a look at it and see what we can do.

**Comment by JanKallman** [comment]
Yes, the metadata.xml file within the package is not copied to external packages. I will remove the meta data attributes from the cells when copying a worksheet to an external workbook for now, until we add support for this.
